src/Domain/LeanCode.DomainModels/Ids/TypedIdConverter.cs

```diff
@@ -34,6 +55,18 @@ public class LongTypedIdConverter<TId> : JsonConverter<TId>
 
     public override void Write(Utf8JsonWriter writer, TId value, JsonSerializerOptions options) =>
         writer.WriteNumberValue(value.Value);
+
+    public override TId ReadAsPropertyName(
+        ref Utf8JsonReader reader,
+        Type typeToConvert,
+        JsonSerializerOptions options
+    ) =>
+        reader.GetString() is string s && long.TryParse(s, out var id)
```

Comment:
Why not [GetInt64](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.getint64?view=net-7.0)?

Reply:
The token value won't be a `Number` here.

Reply:
Good point, sorry